rundeckapp/grails-spa/build.gradle

```diff
@@ -27,7 +27,7 @@ node{
     
     // Version of node to download and install (only used if download is true)
     // It will be unpacked in the workDir
-    version = "18.18.1"
+    version = "18.20.4"
```

Comment:
here the `node.install` updated above could've been referenced. Same in `ui-trellis`